lib/rubocop/cop/registry.rb

```diff
@@ -182,8 +182,12 @@ def names
         cops.map(&:cop_name)
       end
 
+      def cops_for_department(department)
+        cops.select { |cop| cop.department == department.to_sym }
+      end
+
       def names_for_department(department)
-        cops.select { |cop| cop.department == department.to_sym }.map(&:cop_name)
```

Comment:
just moving it into a reusable method. I could make it private as well, but it seems useful as a public one (potentially?)